src/app/course-list/course/component.ts

```diff
@@ -35,11 +38,19 @@ export class CourseComponent {
     this.selectService.toggleCourse(course);
   }
 
+  public hasSelected() {
```

Comment:
I named it that because technically a `course` can't be "selected" right? Rather, courses would have sections that are selected.

I can change it if you think it's better your way though.